bootstrap-salt.sh

```diff
@@ -2663,6 +2669,11 @@ install_debian_deps() {
         __PIP_PACKAGES="${__PIP_PACKAGES} 'requests>=$_PY_REQUESTS_MIN_VERSION'"
     fi
 
+    if [ "$DISTRO_MAJOR_VERSION" -gt 8 ]; then
```

Comment:
Hey, it will only work for Debian version higher than 8... And since currently there is no Debian 9 existing yet, this expression will be false.

By the way, Salt packages build for Debian don't actually need `python-systemd`, they simply work without it.
The systemd service files are little bit different from the ones in Ubuntu packages.

So I think this change is unnecessary.